python/b2gedmntuples_cff.py

```diff
@@ -762,6 +746,10 @@
         tag = cms.untracked.string("tau3Puppi"),
         quantity = cms.untracked.string("userFloat('ak8PFJetsPuppiValueMap:NjettinessAK8PuppiTau3')")
         ),
+     cms.PSet(
+        tag = cms.untracked.string("subjetSumMassSoftDropPuppi"),
+        quantity = cms.untracked.string("? hasUserFloat('subjetSumMassSoftDropPuppi') ? userFloat('subjetSumMassSoftDropPuppi') : -999 ")
+        ),
```

Comment:
can we rename this to something like uncorrSDMassAK8Puppi?
Would be more self-explanatory I think